src/js/game/GameController.js

```diff
@@ -1227,7 +1228,7 @@ class GameController {
         }, this);
       }
     } else {
-      commandQueueItem.failed();
+      commandQueueItem.succeeded();
```

Comment:
why succeed in this case?

Reply:
because failed causes you to lose control of your entity, and this was simply NEVER hit because the if case that corresponds with that else{} was this.levelModel.canPlaceBlock(), which simply did nothing but return true;

🤢